static/css/components/generic-dropper.less

```diff
@@ -88,3 +88,113 @@
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+/* Style for read button and dropper button */
```

Comment:
Is this code supposed to be in the Locate PR?

This may be causing issues with CSS specificity